src/app/features/auth/components/register-form/register-form.component.html

```diff
@@ -0,0 +1,115 @@
+<div class="relative">
```

Comment:
You don't need these kind of wrappers in 99% cases. Your `@Component` decorator has `host` property, where you can pass an object with any html attribute and its' value.
```ts
@Component({
    host: { class: 'relative' }
})
```